examples/source/user-consent/Geolocation-based_Consent_Flow.html

```diff
@@ -75,51 +77,110 @@
 </head>
 <body>
   <!-- ## Setting up amp-geo -->
-  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the present `eea` which includes all countries of the European Economic Area. -->
+  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the iso code `us` which represents the United States, and "preset-eea" which is a [predefined](https://github.com/ampproject/amphtml/blob/master/extensions/amp-geo/0.1/amp-geo-presets.js) country group-->
   <amp-geo layout="nodisplay">
     <script type="application/json">
       {
         "ISOCountryGroups": {
-          "eu": [ "preset-eea" ]
+          "eu": [ "preset-eea" ],
+          "us": [ "us" ]
         }
       }
     </script>
   </amp-geo>
 
+  
   <!-- ## Defining the consent flow -->
   <!--
-    We define a basic consent flow, similar to [this sample](/content/amp-dev/documentation/examples/documentation/Basic_User_Consent_Flow.html). he flow should only trigger for users in the EU, so we set the flag: `"promptIfUnknownForGeoGroup": "eu"`.
+    The flow should rely on amp-geo to determine which config to show based upon the user's location, so we set the flag: `geoOverride`.
+
+    You can learn more about the geoOverride flag [here](https://github.com/ampproject/amphtml/tree/master/extensions/amp-consent/amp-consent.md#geoOverride).
```

Comment:
link to the doc on amp.dev insteac

Reply:
Currently, amp.dev's documentation on amp-consent does not contain references to geoOverride. But we can update that in the next PR. For now I will update this link to point at amp.dev